resources/views/hello.blade.php

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+    <h1>Hello World!</h1>
+</body>
+
+</html>
```

Comment:
不要な改行の削除と、インデントの修正をお願いいたします！
（コードレビュー時は緑背景の部分が修正提案になります。これを参考に修正して再度コミットプッシュし、コメント修正した旨チャットで教えてください！）

```suggestion
<html>
　<body>
    　<h1>Hello World!</h1>
　</body>
</html>
```